src/controller/java/src/chip/devicecontroller/model/InvokeElement.java

```diff
@@ -19,18 +19,45 @@
 
 import java.util.Locale;
 import java.util.Objects;
+import java.util.Optional;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import org.json.JSONException;
+import org.json.JSONObject;
 
 /** An invoke element that should be used for interaction model invoke request and response. */
 public final class InvokeElement {
   private final ChipPathId endpointId, clusterId, commandId;
-  private final byte[] tlv;
+  private final Optional<byte[]> tlv;
+  private final Optional<JSONObject> json;
+  private static Logger logger = Logger.getLogger(InvokeElement.class.getName());
```

Comment:
`logger` can be final -- can it be moved to the top for consistency with other classes?